internal/plugins/helm/plugin.go

```diff
@@ -15,11 +15,17 @@
 package helm
 
 import (
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
 	"sigs.k8s.io/kubebuilder/pkg/plugin"
 )
 
 const pluginName = "helm" + plugin.DefaultNameQualifier
 
+var (
+	supportedProjectVersions = []string{config.Version2, config.Version3Alpha}
+	pluginVersion            = plugin.Version{Number: 1, Stage: plugin.AlphaStage}
```

Comment:
The first release of this plugin should be stable ideally. The default plugin for any project type should never be unstable.
```suggestion
	pluginVersion            = plugin.Version{Number: 1}
```